### ISSUE TITLE
Some scripts do not clear properly from Active Scripts page.

### ISSUE THREAD

**Issue by boozledorf** [danielyxie/bitburner#194]
![Example screenshot](https://i.snag.gy/kDWsPB.jpg)

All instances of wrapper.script have been finished, some for over an hour.  Not sure what the cause is.
wrapper.script is a special script I wrote to be able to short-circuit crime-running scripts that auto repeat for certain lengths of time (or indefinitely).  Its contents are below:

```javascript
/***************
/ wrapper.script
/ Wraps an "unkillable" script
/ accepts a script name and up to 10 arguments to pass to it
/ Creates a prompt, and when the prompt is resolved, kills the "unkillable" script
***************/
script = args.shift();
//Collects up to 10 arguments you wish to pass to the wrapped script.
//If less than 10 arguments are to be passed, the rest are filled with 0s.
for(arg = args.length; arg < 10; arg++) {
    args.push(0);
}
//Run the script with 1 thread and all 10 arguments.
run(script, 1, args[0], args[1], args[2], args[3], args[4], args[5], args[6], args[7], args[8], args[9]);
//Prompt the user to prematurely end the script.
//SCRIPT EXECUTION PAUSES UNTIL THE PROMPT IS RESOLVED.
prompt("Kill the script?  Both options do the same.");
//Once the prompt is resolved, forcibly kill the script.
//The script name, the host name, and all passed arguments must be included.
kill(script, getHostname(), args[0], args[1], args[2], args[3], args[4], args[5], args[6], args[7], args[8], args[9]);
```

It's possible that this bug is caused by modifying the `args` variable, but I'm not sure.

**Comment by danielyxie** [comment]
This is because you are changing the `args` array. This will be fixed in the next update

**Comment by danielyxie** [comment]
Fixed in v0.34.5
